nifi-extension-bundles/nifi-couchbase-bundle/nifi-couchbase-services-api-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,21 @@
+nifi-couchbase-services-api-nar
+Copyright 2014-2024 The Apache Software Foundation
+
```

Comment:
there is also reactive-streams in the build but not referenced.  It is MIT licensed which means it needs to be copied into a LICENSE file and carried forward as appropriate to the assembly LICENSE too if not already.

https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.4/LICENSE